src/components/MoviesList.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import MoviesItem from './MoviesItem';
+import MovieContextMenu from './MovieContextMenu';
+import moviesData from '../api/movies.json';
+import '../styles/MoviesList.scss';
+
+function MoviesList({ showMovieModal, handleMovieClick }) {
+  return (
+    <div className="movies-wrapper">
+      <div className="movies-count">
+        <strong>{moviesData.length}</strong> movies found
+      </div>
+
+      <ul className="movies-list">
+        {moviesData.map(movie => {
+          return (
+            <li className="movies-item" key={movie.title}>
+              <MovieContextMenu movie={movie} showMovieModal={showMovieModal} />
+              <MoviesItem movie={movie} handleMovieClick={handleMovieClick} />
```

Comment:
To my mind it it will be better to rename `handleMovieClick={` to `movieClickHandler={` (JFYI you can skip this comment since it is related to a taste)

Reply:
Fixed